Packs/Binalyze/Integrations/BinalyzeAIR/BinalyzeAIR.py

```diff
@@ -15,11 +13,32 @@ def test_api(self):
             url_suffix='/api/public/endpoints?filter[organizationIds]=0'
         )
 
-    def air_acquire(self, hostname: str, profile: str, case_id: str, organization_id: int) -> Dict[str, str]:
+    def get_profile_id(self, profile: str, organization_id: Optional[int]) -> str:
+        '''Gets the profile ID based on the profile name and organization ID by making a GET request to the
+        '/api/public/acquisitions/profiles' endpoint.
+        Args:
+        profile (str): The name of the profile to query.
+        organization_id (int): The organization ID associated with the profile.
+        Returns:
+        str: The profile ID obtained from the API response.
+        Raises:
+        DemistoException: If there is an error making the HTTP request or processing the API response.
+        '''
+        preset_profiles = ["browsing-history", "compromise-assessment", "event-logs", "full", "memory-ram-pagefile", "quick"]
+        if profile in preset_profiles:
+            return profile
+        else:
+            return self._http_request(
+                method='GET',
+                url_suffix=f'/api/public/acquisitions/profiles?filter[name]='
+                           f'{profile}&filter[organizationIds]={organization_id}')['result']['entities'][0]['_id']
+
+    def air_acquire(self, hostname: str, profile: str, case_id: str, organization_id: Optional[int]) -> Dict[Any, str]:
```

Comment:
You have a type hint mismatch your return type is Dict[Any, str], but in line 122 the return type is Dict[str, Any], please recheck your type hint.

Reply:
done